### PR TITLE
Fixes#62

### LINKED ISSUE #62: size of about card is changed, get it back to previous one.
screenshot from 2019-02-04 10-44-38

### PR DESCRIPTION
**Pull request by Archies11**
# Description

Changed size of about card.

Fixes #62

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [] New feature (non-breaking change which adds functionality)

## Live Gh-Pages Link
[Live Link]https://competent-minsky-9849cd.netlify.com/

## Screenshot of the change

capture

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Anupam-dagar** [comment]
The changes done are quite hardcoded, can you take a look around the breaking css commit and see what really happened to the about card. This looks like forcing the card to be in a position rather than it being making a proper use of css properties.

File: index.html [hunk 1]
```diff
@@ -77,14 +77,14 @@
           </div><!-- /.col -->
         </div><!-- /.row -->
         <div class="row justify-content-center ">
-          <div class="card shadow-lg rounded " style="width: 65rem;">
```
Comment (Anupam-dagar) [inline index.html]: rem is a flexible unit, these all px values at each div are just holding the card together which might break anytime something is worked upon the about card.
Try to look for the cause of the about card getting shrinked.

**Comment by Archies11** [comment]
I did find the root cause, the bootstrap css file used for the gallery when commented out, leads to restoration of old font sizes and styles but I am unable to understand why doesnt the bootstrap file get overidden by the below mentioned css files since the code gets executed sequentially?

**Comment by Anupam-dagar** [comment]
HTML, CSS aren't like C, C++ or any other programming languages. They at first place aren't programming languages. Coming to the css properties getting overriden, there is `! important` which is used to override the property if anywhere else used. Bootstrap framework make use of it many places. So I suppose the gallery implementation is itself wrong. Let me see if there are any destructive changes there in that case I need to revert it else you should first fix the gallery which will automatically lead to fixing of these issues.

**Comment by Archies11** [comment]
> HTML, CSS aren't like C, C++ or any other programming languages. They at first place aren't programming languages. Coming to the css properties getting overriden, there is `! important` which is used to override the property if anywhere else used. Bootstrap framework make use of it many places. So I suppose the gallery implementation is itself wrong. Let me see if there are any destructive changes there in that case I need to revert it else you should first fix the gallery which will automatically lead to fixing of these issues.

ok, on it. @Anupam-dagar

**Comment by Archies11** [comment]
Pr that solves this issue : https://github.com/opencodeiiita/CodeFemme/pull/76

**Pull request closed without merging**